service/src/main/java/io/camunda/service/UserServices.java

```diff
@@ -43,8 +44,8 @@ public UserServices withAuthentication(final Authentication authentication) {
     return new UserServices(brokerClient, searchClient, transformers, authentication);
   }
 
-  public CompletableFuture<UserRecord> createUser(final CreateUserRequest request) {
-    return sendBrokerRequest(
+  public CompletableFuture<BrokerResponse<UserRecord>> createUser(final CreateUserRequest request) {
+    return sendBrokerRequestWithFullResponse(
```

Comment:
❓ Why this change ? In `ResponseMapper` you are using only the response object, and not the full `BrokerResponse`. Am I missing something ?

Reply:
That's just my lack of knowledge of the REST API. I copied it from another request 😄 I'll change it.